react-table/src/Paging.tsx

```diff
@@ -32,8 +32,12 @@ export default function (props: IProps) {
     const [pages, setPages] = React.useState<number[]>([]);
 
     const nPages = 7;
-    const minPg = React.useMemo(() => Math.min(...pages), [pages]);
-    const maxPg = React.useMemo(() => Math.max(...pages), [pages]);
+    const minPg = React.useMemo(() => {
+        const min = Math.min(...pages);
+        if (isFinite(min)) return min;
+        else return 0;
+    }, [pages]);
+    const maxPg = React.useMemo(() => Math.max(...pages, 0), [pages]);
```

Comment:
```suggestion
    const maxPg = React.useMemo(() => {
        const max = Math.max(...pages);
         if (isFinite(max) && !isNaN(max)) return max;
         return 0
    }, [pages]);
```

This should cover the `NaN` case and any positive infinite numbers as well